File: src/main/java/com/lab/reservation/entity/Lab.java
package com.lab.reservation.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@TableName("lab")
public class Lab extends BaseEntity {
    private String name;
    private String location;
    private Long managerId;
    private String description;
    private Integer status;
}

File: frontend/src/types/lab.ts
/** 实验室（entity/Lab.java）。 */
export interface Lab {
  id: number
  name: string
  location?: string
  managerId?: number
  description?: string
  /** 0 禁用 / 1 启用 */
  status?: number
  createdAt?: string
  updatedAt?: string
}
